### PR TITLE
Various fixups

### PR DESCRIPTION
**Pull request by nikicc**

### CODE REVIEW

File: orangecontrib/text/widgets/utils/widgets.py [hunk 1]
```diff
@@ -473,13 +473,13 @@ def __init__(self, widget, model_format, provider_format,
 
         self.model_path = None
         layout = QtGui.QHBoxLayout(self)
-        layout.setContentsMargins(0, 0, 0, 0)
+        layout.setSpacing(0)
+        layout.setMargin(0)
```
Comment (kernc) [inline orangecontrib/text/widgets/utils/widgets.py]: Obsolete in Qt5. Says to use `setContentsMargins()` instead.

Comment (nikicc) [inline orangecontrib/text/widgets/utils/widgets.py]: Fixed.

**Comment by nikicc** [comment]
@ajdapretnar I think this is ready for a review.

**Comment by ajdapretnar** [comment]
Finally! :grin:

**Comment by nikicc** [comment]
@ajdapretnar better late than never 😉

**Pull request merged**